includes/wikia/services/HubService.class.php

```diff
@@ -61,6 +81,31 @@ public static function getCurrentWikiaVerticalName() {
 	}
 
 	/**
+	 * Get canonical vertical name for given cityId.
+	 * For Lifestyle and Gaming return their names.
+	 * For rest of values return Entertainment.
+	 * @param $cityId
+	 * @return Boolean|String
+	 */
+	public static function getVerticalNameForComscore($cityId) {
+		$verticalId = WikiFactoryHub::getInstance()->getVerticalId($cityId);
+		$canonicalVerticalId = self::getCanonicalVerticalID($verticalId);
+
+		switch ($canonicalVerticalId) {
```

Comment:
getVerticalId doesn't return a field which maps to the CATEGORY_ID

See here: https://github.com/Wikia/app/blob/dev/extensions/wikia/WikiFactory/Hubs/WikiFactoryHub.php#L22

And actually, I'm not sure if we have that mapping defined anywhere, but we should probably have one.  We need a mapping of vertical to "legacy category" for comscore, but I think we also need to fall back on the old category if the vertical_id is 0.  

I think we also need to double check the names, as they are different than what is defined in HubService. 

Also, there's a fallback in the getCategoryInfoForCity() function that looks for a comscore override tag, and I think we need to preserve that logic (line 102 of HubService)